src/device/wg_controller.rs

```diff
@@ -194,10 +194,18 @@ async fn consolidate_wg_peers<
                 if let (Some(sk), Some(mesh_ip)) =
                     (session_keeper, requested_peer.peer.allowed_ips.first())
                 {
+                    // TODO this should be acommodatedto IPv6
+                    let ip4 = {
+                        match mesh_ip.ip() {
+                            IpAddr::V4(ip4) => ip4,
+                            IpAddr::V6(_ip6) => Err(Error::BadAllowedIps)?,
```

Comment:
error doesn't really make sense, maybe panic as well? or if this can happen, skip this peer(continue or smth), printing warning?

Reply:
Fixed